module/projectAttributes/helper.js

```diff
@@ -0,0 +1,210 @@
+const projectAttributesQueries = require(DB_QUERY_BASE_PATH + '/projectAttributes')
+const entitiesService = require(GENERICS_FILES_PATH + '/services/entity-management')
+
+module.exports = class ProjectAttributesHelper {
+	/**
+	 * create project attributes
+	 * @method
+	 * @name create
+	 * @returns {Object} .
+	 */
+	static async create() {
+		try {
+			// Get the default project attributes
+			let getProjectAttribute = CONSTANTS.common.DEFAULT_ATTRIBUTES
+			//Getting roles from the entity service
+			let userRoleInformation = await entitiesService.userExtensionDocuments(
+				{
+					status: CONSTANTS.common.ACTIVE_STATUS.toUpperCase(),
+				},
+				['title', 'code']
+			)
+
+			if (!userRoleInformation.success) {
```

Comment:
@praveenKDass return proper error messages. do it every where, refer existing code